src/environment/environment.uat.ts

```diff
@@ -1,6 +1,7 @@
-import { EnviromentFile } from './models/environment.model';
+import { EnvironmentFile } from './models/environment.model';
 
-export const environment: EnviromentFile = {
+export const environment: EnvironmentFile = {
   isRemote: true,
   remoteSettingsUrl: 'assets/config/environmentUat.json',
+  journalEndPoint: 'https://vulv731rce.execute-api.eu-west-1.amazonaws.com/default'
 };
```

Comment:
If remote is true, the config provider won't read any of the data in this file apart from the remoteSettingsUrl.

The journalEndPoint needs to be in the assets/config/environmentUat.json file

Reply:
Fixed